e2e_samples/parking_sensors/scripts/deploy_infrastructure.sh

```diff
@@ -204,40 +200,28 @@ validate_password() {
 echo "Creating Service Principal (SP) for access to ADLA Gen2 used in Databricks mounting"
 
 # ###########################
-###if there is an hyphen retry and recreate the SP with a new password
+###if there is an hyphen in the first position replace by R
```

Comment:
wording is a bit confusing in the comment
you mean replace with the letter R correct?
and should the comment be here, or in the section above (the function itself)?